Octokit/Models/Response/User.cs

```diff
@@ -23,6 +27,13 @@ public User(string avatarUrl, string bio, string blog, int collaborators, string
         /// </summary>
         public bool SiteAdmin { get; protected set; }
 
+        /// <summary>
+        /// LDAP Binding (GitHub Enterprise only)
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Dn")]
+        [SuppressMessage("Microsoft.Naming", "CA1709:IdentifiersShouldBeCasedCorrectly", MessageId = "Dn")]
+        public string LdapDn { get; protected set; }
```

Comment:
> I also wasnt sure if I could set what the "api name" of a given property is

Yes, it's possible via the `Parameter` attribute. Here's [one example](https://github.com/octokit/octokit.net/blob/master/Octokit/Models/Request/EditRepositoryHook.cs#L40).